### DISCUSSION TITLE
Issues with GitHub Pages?

### DISCUSSION THREAD

**Discussion by DJBenson** [Koenkk/zigbee2mqtt#22922] [General]
I'm currently unable to access any sites on GitHub pages (including the Z2M site and the darkxst firmware builder). The block of IP's associated (185.199.10[9-12].153 are all inaccessble and a traceroute shows lots of lost packets beyond my network.

I've done a couple of remote ping and traceroutes and they are showing varying levels of degredation depending on country.

I've asked on a UK-based group and getting a small number of confirmations - any UK users here having the same issue (possibly on BT-based networks).

**Comment by Vospe1** [comment]
Yes, having the same issues. However it is not for all GitHub pages, but Z2M is definitely impacted. 
I am also UK (and BT) based.

**Reply by DJBenson** [reply]
Thanks for confirming. I have a handful of cases with other people on Plusnet/BT being impacted so I've raised it with Plusnet.

**Comment by DJBenson** [comment]
It seems to have cleared now so I'll close this.
